src/content/api/accounts.mdx

```diff
@@ -0,0 +1,263 @@
+---
+title: Accounts
+description: Account model and related endpoints
+draft: true
+nav:
+  path: API
+  order: 7
+---
+import Properties from '../../components/Properties.astro';
+import Property from '../../components/Property.astro';
+import Endpoint from '../../components/Endpoint.astro';
+import CodeGroup from '../../components/CodeGroup.astro';
+
+An Account represents a permission boundary around Centrapay resources.
+Accounts can have [API Keys](/api/api-keys/) and [Account Memberships](/api/account-memberships/) which grant access to the resources.
+
+Accounts are classified as either "individual" or "org". Individual accounts
+can only have a single member and Centrapay users can only be a member of a single
+individual account.
+
+## Account Model
```

Comment:
The media uploads page has `Model` but that doesn't work very well for this page as we have two models. Any suggestions here. We could just have `Account` and `Subscription`

Reply:
Maybe we should name the models rather than have just model. Do we even need a model as it is just an array. Could we just add to the account model?

Reply:
I'm not sure what you mean here. At the moment we have two named models - `Account Model` and `Subscription Model`. But these could be changed to `Account` and `Subscription`. 

Subscriptions is just an array but we have other use cases where the sub-model is much larger and so having a single model could get messy

Reply:
Sorry I read over my comment and it was a bit confusing. I was esentiallly agreeing with you to just have the name of the model and leave off the model.
The suggestion for the subscription model was just for this particular scenario.

Reply:
Sweet as - I think `Account Model` reads better in the navigation bar 🤔

Reply:
@AndyClifford suggestion: We have `Model` as a H2 and then H3 `Account` and `Subscription`